Simplenote.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -32,9 +32,9 @@
         "package": "XCUITestHelpers",
         "repositoryURL": "https://github.com/Automattic/XCUITestHelpers",
         "state": {
-          "branch": "trunk",
+          "branch": null,
           "revision": "47d688b0b2a6356361a16836c4d23f12d6643a73",
-          "version": null
+          "version": "0.1.0"
```

Comment:
This JSON format is horrendous. `branch` is now `null` because there is a `version` set. You may notice that before it was the opposite. The `revision` is the same because 0.1.0 was tagged on top of the tip of the package `trunk` to which this project was pointing at the time.